task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -40,7 +42,6 @@ spec:
       computeResources:
         limits:
           memory: 4Gi
-          cpu: 2
```

Comment:
Do not remove the max CPU. We want to get rid of that eventually be currently we do have a limitRange which specify the default CPU limit for any containers that do not set it. If you remove it here, this task will end up using the default at the namespace level. Put it back for time being, we will remove it in a follow up change once we finish ongoing change in toolchain-host-operator to remove default CPU limits from LimitRange.

Reply:
look good now, I cannot resolve this thread, looks like I lack some permission on this repository...

Reply:
Thanks! Let me resolve it.